tables/TaQL/test/tTableGram.out

```diff
@@ -64,7 +64,7 @@ select ab from tTableGram_tmp.tab where ab==2**1**2 || ab==-2**-1*8/-2*3
 1 selected columns:  ab
  2
  6
-select ab,ac,af from tTableGram_tmp.tab where lower(af) == regex("v[01279]")
+select ab,ac,af from ../../TaQL/./test/tTableGram_tmp.tab where lower(af) == regex("v[01279]")
```

Comment:
This seems to be unrelated

Reply:
Indeed it is testing those, but I'll undo these changes in this PR.